network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html

```diff
@@ -1,77 +1,33 @@
 {% extends "pages/base.html" %}
-{% load wagtailimages_tags i18n %}
+{% load wagtailcore_tags %}
 
+{% comment %}
+  TODO: Breadcrumbs, Factory generation for authors
+{% endcomment %}
 {% block content%}
-    <div class="container">
-
-        <div
-        class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center">
-            {% comment %} <div class="col-sm-6 mb-3 mb-sm-0">
-                {% image page.hero_image fill-1240x625 %}
-            </div> {% endcomment %}
-
-            <div class="col-sm-10">
-
-                <h1 class="h1-heading">
-                    {{ page.title }}
-                </h1>
-
-                <div class="body-large mb-4">
-                    {% if page.subtitle %}
-                        {{ page.subtitle }}
-                    {% endif %}
-                </div>
 
-                <div class="body-large mb-4">
+  {% include "./fragments/publication_hero.html" %}
 
-                    {% if page.secondary_subtitle %}
-                        {{ page.secondary_subtitle }}
-                        {% endif%}
-                    </div>
-                    {% if page.publication_date %}
-                        {{ page.publication_date }}
-                    {% endif %}
+  {% if not page.is_chapter_page %}
+    {# Publication Page #}
+    {% include "fragments/publication_table_of_contents.html" %}
+  {% else %}
+    {# Chapter page #}
+    {% include "fragments/chapter_table_of_contents.html" %}
+  {% endif %}
```

Comment:
A nit, but: if there's a property `is_x`  and there is if/else logic, using `if (is_x) { doForX } else { doForNotX }` is easier to read than `if (!is_x) { doForNotX } else { doForX }`.